design/one-pager-pluggable-secret-stores.md

```diff
@@ -154,47 +153,29 @@ message DeleteOptions {
 
 A _Secret Store Plugin_:
 
-- Will be a binary packaged as a container image.
-- Should run a gRPC server listening at Unix domain socket.
-- May support one or more Secret Stores.
+- Will be a k8s deployment behind a service.
+- Should run a gRPC server.
+- Should support one Secret Store.
 - Should deploy CRD(s) for supported store configurations at startup.
-- Will run as a sidecar next to Crossplane and Provider containers.
+- Should have server certificates signed by the kubernetes cluster root CA.
+- Should require and verify client certificates.
 
-#### Plugin Binary vs Secret Store Implementation
+### Deployment
```

Comment:
Could you add some details for **Packaging** as well, e.g. we will define a new package type called `SecretStorePlugin` etc.